### DISCUSSION TITLE
Alt-Tab switcher: Gnome-Style Thumbnails with Icons

### DISCUSSION THREAD

**Discussion by Nekkowe** [linuxmint/discussions#360] [Ideas]
Hello!

Currently, the "Thumbnails only" switcher style can make it hard to distinguish multiple windows at a glance.
Especially when several opened programs all use Dark Mode, they can start to look very similar.
The "Icons and Thumbnails" style makes it easier to distinguish, but unfortunately, it hides the thumbnails entirely for all but one window, so you can't look at them side-by-side at all anymore...

It would be nice to have view of both the icons and the thumbnails for all open programs at the same time.

I think it would help a lot if the "Thumbnails only" still displayed the icon in some smaller way, either in the corner of the thumbnail like this:
![image](https://github.com/linuxmint/discussions/assets/28980049/83567cf3-cde1-4b31-adcd-3e4e5822bb44)
...or even just as a small icon to the left of the window title label.

**Comment by 100savage** [comment]
If you go to system settings ->Windows -> the top will have a tab called "Alt-Tab"
You can select "Icons and thumbnail" from the dropdown and get the same affect.

**Reply by Nekkowe** [reply]
Like I said in the first post,

>  The "Icons and Thumbnails" style makes it easier to distinguish, **but unfortunately, it hides the thumbnails entirely for all but one window,** so you can't look at them side-by-side at all anymore...

It's not the effect I'm looking for at all - it shows one big icon for each window, and then the thumbnail for _only_ the currently selected window.
